### ISSUE TITLE
Error while uploading to AppStore

### ISSUE THREAD

**Issue by ThomasDhaen** [ge-org/multiplatform-swiftpackage#19]
Hey

I have a setup where, for a new release, I run `./gradlew createXCFramework` and commit the .xcframework on my release branch.
This works pretty well, however, when building an IPA and trying to submit to the store, I get the following error:

```
[Transporter Error Output]: ERROR ITMS-90171: "Invalid Bundle Structure - The binary file 'MyApp.app/Frameworks/mysdk.framework/mysdk' is not permitted. Your app can't contain standalone executables or libraries, other than a valid CFBundleExecutable of supported bundles. Refer to the Bundle Programming Guide at https://developer.apple.com/go/?id=bundle-structure for information on the iOS app bundle structure."
```

build.gradle:
```
multiplatformSwiftPackage {
    swiftToolsVersion("5.3")
    targetPlatforms {
        iOS { v("12") }
    }
    outputDirectory(File(".."))
}
```

Any idea what could cause this?

**Comment by ge-org** [comment]
Hi @ThomasDhaen,

my guess is that the `mysdk.framework` file is being copied into the bundle during the build. However, that's not allowed by Apple.

Please check your build settings in Xcode to make sure the framework is not part of the bundle. You might want to look into the following places:
- in the 'project navigator' click on the framework file and then make sure that no checkbox is ticked in the 'target membership' section of the 'file inspector'
- make sure the 'copy bundle resources' in the 'build phases' tab of the target settings does not contain the framework file

Please let me know if this resolves your issue

**Comment by ge-org** [comment]
Dis you manage to fix the problem?
It could also be related to a limitation of SPM that Apple is currently investigating.

https://forums.swift.org/t/swift-packages-in-multiple-targets-results-in-this-will-result-in-duplication-of-library-code-errors/34892?page=2

**Comment by ThomasDhaen** [comment]
Hey @ge-org thanks for sharing.
We've checked these points and both are not the case for our project.

I'm going to make a sample project and see if I can reproduce it, will post back later today or tomorrow!

**Comment by ThomasDhaen** [comment]
We found what caused this error: 
Our build.grade file included plugin `"org.jetbrains.kotlin.native.cocoapods"` (although we don't use it anymore).

This seemed to be putting the binary in an archive and causing the invalid bundle structure error.
We've removed this plugin and this has resolved it for us.

With cocoapods plugin:
```
file mysdk
mysdk: current ar archive random library
```

Without cocoapods plugin:
```
file mysdk
mysdk: Mach-O 64-bit dynamically linked shared library arm64
```
